mvsnet/test.py

```diff
@@ -24,20 +24,20 @@
 tf.app.flags.DEFINE_string('output_dir', None,
                            """Path to data to dir to output results""")
 tf.app.flags.DEFINE_string('model_dir',
-                           'gs://mvs-training-mlengine/trained-models/06-04-2019/',
+                           'gs://mvs-training-mlengine/dd7_alpha_0_25_epsilon_linear_0_005_lr_0_005_4gpu/models/',
```

Comment:
so this model doesn't need to be in the trained-models folder?

Reply:
Good question. This model is just being used for testing / benchmarking. In inference.py we still have the model stored in the trained-models directory